src/pages/vagrant/downloads/index.tsx

```diff
@@ -1,9 +1,30 @@
+import GetStartedCard from 'components/get-started-card'
 import vagrantData from 'data/vagrant.json'
 import { ProductData } from 'types/products'
 import ProductDownloadsView from 'views/product-downloads-view'
 import { generateGetStaticProps } from 'views/product-downloads-view/server'
 
+function VagrantDownloadsMerchandisingSlot() {
+	return (
+		<GetStartedCard
```

Comment:
I'm going to rename this component to just 'Callout' in a follow up PR, as it's used on the vagrant landing page multiple times, not just in the getting started context.